easyapplybot.py

```diff
@@ -262,70 +281,174 @@ def click_button(self, xpath):
-                                #print("Error Found")
-                                #print(element.get_attribute('class'))
-                                button = None
-                                break
+                                log.warning("Warning message received: %s", text)
+                                log.info("Attempting to resolve by finding test questions")
+
+                                #TODO these questions will need to be logged so that way, individuals can look through the logs and add them at the end of an application run.
+                                #Required question expects an answer. Search through possible questions/answer combos
+                                if is_present(testLabel_locator) and attemptQuestions:
+                                    for testLabelElement in self.browser.find_elements(testLabel_locator[0],
+                                                                              testLabel_locator[1]):
+                                        try:
+                                            log.info("Found test element %s", testLabel_locator)
+                                            text = testLabelElement.text
+                                            log.info("test element text: %s", text)
+                                            #assuming this question is asking if I am authorized to work in the US
+                                            if ("Are you" in text and "authorized" in text) or ("Have You" in text and "eduation" in text):
+                                                #Be sure to find the child element of the current test question section
+                                                yesRadio = testLabelElement.find_element(By.XPATH, yes_locator[1])
+                                                time.sleep(1)
+                                                log.info("Attempting to click the radio button for %s", yes_locator)
+                                                self.browser.execute_script("arguments[0].click()", yesRadio)
+                                                log.info("Clicked the radio button %s", yes_locator)
+
+                                            #assuming this question is asking if I require sponsorship
+                                            if "require" in text and "sponsorship" in text:
+                                                noRadio = testLabelElement.find_element(By.XPATH, no_locator[1])
+                                                time.sleep(1)
+                                                log.info("Attempting to click the radio button for %s", no_locator)
+                                                self.browser.execute_script("arguments[0].click()", noRadio)
+                                                log.info("Clicked the radio button %s", no_locator)
+
+                                            # assuming this question is asking if I require sponsorship
```

Comment:
The comment should refer to degree/bachelor and not sponsorhip

Reply:
Thanks! Fixed